packages/composables/src/useUser.ts

```diff
@@ -151,6 +155,10 @@ export function useUser(): UseUserReturn {
   const salutation: Ref<Schemas["Salutation"] | null> = ref(null);
   const user = computed(() => _user.value);
 
+  const canSyncWishlist = computed(
+    () => isLoggedIn.value && !isGuestSession.value,
+  );
```

Comment:
that is not what this condition tells us. Also both `isLoggedIn` and `isGuestSession` are already exposed by the `useUser` let's revert changes in this file and check it in actual wishlist

Reply:
moved again